### ISSUE TITLE
Error building using gcc go 1.13.1

### ISSUE THREAD

**Issue by micwoj92** [twmb/murmur3#11]
210dd316651419f3403ae3443b8724e33538eeec
`go version go1.18 gccgo (GCC) 13.1.1 20230429 linux/amd64`

```
go build
# github.com/twmb/murmur3
./murmur128_amd64.s: Assembler messages:
./murmur128_amd64.s:4: Error: no such instruction: `text ·SeedSum128(SB),$0-56'
./murmur128_amd64.s:5: Error: junk `(FP)' after expression
./murmur128_amd64.s:5: Error: junk `(FP)' after expression
./murmur128_amd64.s:5: Error: operand size mismatch for `movq'
./murmur128_amd64.s:6: Error: junk `(FP)' after expression
./murmur128_amd64.s:6: Error: junk `(FP)' after expression
./murmur128_amd64.s:6: Error: operand size mismatch for `movq'
./murmur128_amd64.s:7: Error: junk `(FP)' after expression
./murmur128_amd64.s:7: Error: junk `(FP)' after expression
./murmur128_amd64.s:7: Error: operand size mismatch for `movq'
./murmur128_amd64.s:8: Error: junk `(FP)' after expression
./murmur128_amd64.s:8: Error: junk `(FP)' after expression
./murmur128_amd64.s:8: Error: operand size mismatch for `movq'
./murmur128_amd64.s:9: Error: junk `(FP)' after expression
./murmur128_amd64.s:9: Error: operand type mismatch for `lea'
./murmur128_amd64.s:13: Error: no such instruction: `text ·Sum128(SB),$0-40'
./murmur128_amd64.s:14: Error: operand size mismatch for `xor'
./murmur128_amd64.s:15: Error: operand size mismatch for `xor'
./murmur128_amd64.s:16: Error: junk `(FP)' after expression
./murmur128_amd64.s:16: Error: junk `(FP)' after expression
./murmur128_amd64.s:16: Error: operand size mismatch for `movq'
./murmur128_amd64.s:17: Error: junk `(FP)' after expression
./murmur128_amd64.s:17: Error: junk `(FP)' after expression
./murmur128_amd64.s:17: Error: operand size mismatch for `movq'
./murmur128_amd64.s:18: Error: junk `(FP)' after expression
./murmur128_amd64.s:18: Error: operand type mismatch for `lea'
./murmur128_amd64.s:22: Error: no such instruction: `text ·SeedStringSum128(SB),$0-48'
./murmur128_amd64.s:23: Error: junk `(FP)' after expression
./murmur128_amd64.s:23: Error: junk `(FP)' after expression
./murmur128_amd64.s:23: Error: operand size mismatch for `movq'
./murmur128_amd64.s:24: Error: junk `(FP)' after expression
./murmur128_amd64.s:24: Error: junk `(FP)' after expression
./murmur128_amd64.s:24: Error: operand size mismatch for `movq'
./murmur128_amd64.s:25: Error: junk `(FP)' after expression
./murmur128_amd64.s:25: Error: junk `(FP)' after expression
./murmur128_amd64.s:25: Error: operand size mismatch for `movq'
./murmur128_amd64.s:26: Error: junk `(FP)' after expression
./murmur128_amd64.s:26: Error: junk `(FP)' after expression
./murmur128_amd64.s:26: Error: operand size mismatch for `movq'
./murmur128_amd64.s:27: Error: junk `(FP)' after expression
./murmur128_amd64.s:27: Error: operand type mismatch for `lea'
./murmur128_amd64.s:31: Error: no such instruction: `text ·StringSum128(SB),$0-32'
./murmur128_amd64.s:32: Error: operand size mismatch for `xor'
./murmur128_amd64.s:33: Error: operand size mismatch for `xor'
./murmur128_amd64.s:34: Error: junk `(FP)' after expression
./murmur128_amd64.s:34: Error: junk `(FP)' after expression
./murmur128_amd64.s:34: Error: operand size mismatch for `movq'
./murmur128_amd64.s:35: Error: junk `(FP)' after expression
./murmur128_amd64.s:35: Error: junk `(FP)' after expression
./murmur128_amd64.s:35: Error: operand size mismatch for `movq'
./murmur128_amd64.s:36: Error: junk `(FP)' after expression
./murmur128_amd64.s:36: Error: operand type mismatch for `lea'
./murmur128_amd64.s:45: Error: no such instruction: `text sum128internal<>(SB),$0'
./murmur128_amd64.s:46: Error: operand type mismatch for `movq'
./murmur128_amd64.s:47: Error: operand type mismatch for `movq'
./murmur128_amd64.s:49: Error: operand size mismatch for `movq'
./murmur128_amd64.s:53: Error: operand size mismatch for `xor'
./murmur128_amd64.s:56: Error: operand size mismatch for `cmp'
./murmur128_amd64.s:58: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:58: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:58: Error: operand size mismatch for `movq'
./murmur128_amd64.s:59: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:59: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:59: Error: operand size mismatch for `movq'
./murmur128_amd64.s:62: Error: operand size mismatch for `imul'
./murmur128_amd64.s:63: Error: operand size mismatch for `imul'
./murmur128_amd64.s:68: Error: operand size mismatch for `imul'
./murmur128_amd64.s:69: Error: operand size mismatch for `imul'
./murmur128_amd64.s:71: Error: operand size mismatch for `xor'
./murmur128_amd64.s:73: Error: operand size mismatch for `add'
./murmur128_amd64.s:74: Error: operand size mismatch for `xor'
./murmur128_amd64.s:76: Error: junk `(R12)(R12*4)' after expression
./murmur128_amd64.s:76: Error: operand type mismatch for `lea'
./murmur128_amd64.s:78: Error: operand size mismatch for `add'
./murmur128_amd64.s:79: Error: junk `(R13)(R13*4)' after expression
./murmur128_amd64.s:79: Error: operand type mismatch for `lea'
./murmur128_amd64.s:84: Error: operand size mismatch for `movq'
./murmur128_amd64.s:88: Error: operand size mismatch for `xor'
./murmur128_amd64.s:123: Error: no such instruction: `movbqzx 14(SI)(R10*1),AX'
./murmur128_amd64.s:127: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:127: Error: operand size mismatch for `mov'
./murmur128_amd64.s:132: Error: no such instruction: `movbqzx 12(SI)(R10*1),AX'
./murmur128_amd64.s:136: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:136: Error: operand size mismatch for `mov'
./murmur128_amd64.s:137: Error: operand size mismatch for `or'
./murmur128_amd64.s:141: Error: no such instruction: `movbqzx 10(SI)(R10*1),AX'
./murmur128_amd64.s:145: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:145: Error: operand size mismatch for `mov'
./murmur128_amd64.s:149: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:149: Error: operand size mismatch for `mov'
./murmur128_amd64.s:152: Error: operand size mismatch for `imul'
./murmur128_amd64.s:154: Error: operand size mismatch for `imul'
./murmur128_amd64.s:155: Error: operand size mismatch for `xor'
./murmur128_amd64.s:158: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:158: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:158: Error: operand size mismatch for `movq'
./murmur128_amd64.s:162: Error: no such instruction: `movbqzx 6(SI)(R10*1),AX'
./murmur128_amd64.s:166: Error: junk `(SI)(R10*1)' after expression
./murmur128_amd64.s:166: Error: operand size mismatch for `mov'
./murmur128_amd64.s:171: Error: no such instruction: `movbqzx 4(SI)(R10*1),AX'
./murmur128_amd64.s:175: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:175: Error: operand size mismatch for `mov'
./murmur128_amd64.s:176: Error: operand size mismatch for `or'
./murmur128_amd64.s:180: Error: no such instruction: `movbqzx 2(SI)(R10*1),AX'
./murmur128_amd64.s:184: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:184: Error: operand size mismatch for `mov'
./murmur128_amd64.s:188: Error: junk `(R10*1)' after expression
./murmur128_amd64.s:188: Error: operand size mismatch for `mov'
./murmur128_amd64.s:191: Error: operand size mismatch for `imul'
./murmur128_amd64.s:193: Error: operand size mismatch for `imul'
./murmur128_amd64.s:194: Error: operand size mismatch for `xor'
./murmur128_amd64.s:197: Error: operand size mismatch for `xor'
./murmur128_amd64.s:198: Error: operand size mismatch for `xor'
./murmur128_amd64.s:200: Error: operand size mismatch for `add'
./murmur128_amd64.s:201: Error: operand size mismatch for `add'
./murmur128_amd64.s:204: Error: operand size mismatch for `movq'
./murmur128_amd64.s:205: Error: operand size mismatch for `movq'
./murmur128_amd64.s:210: Error: operand size mismatch for `xor'
./murmur128_amd64.s:211: Error: operand size mismatch for `xor'
./murmur128_amd64.s:213: Error: operand type mismatch for `movq'
./murmur128_amd64.s:215: Error: operand size mismatch for `imul'
./murmur128_amd64.s:216: Error: operand size mismatch for `imul'
./murmur128_amd64.s:218: Error: operand size mismatch for `movq'
./murmur128_amd64.s:219: Error: operand size mismatch for `movq'
./murmur128_amd64.s:224: Error: operand size mismatch for `xor'
./murmur128_amd64.s:225: Error: operand size mismatch for `xor'
./murmur128_amd64.s:227: Error: operand type mismatch for `movq'
./murmur128_amd64.s:229: Error: operand size mismatch for `imul'
./murmur128_amd64.s:230: Error: operand size mismatch for `imul'
./murmur128_amd64.s:232: Error: operand size mismatch for `movq'
./murmur128_amd64.s:233: Error: operand size mismatch for `movq'
./murmur128_amd64.s:238: Error: operand size mismatch for `xor'
./murmur128_amd64.s:239: Error: operand size mismatch for `xor'
./murmur128_amd64.s:241: Error: operand size mismatch for `add'
./murmur128_amd64.s:242: Error: operand size mismatch for `add'
./murmur128_amd64.s:244: Error: operand size mismatch for `movq'
./murmur128_amd64.s:245: Error: junk `(BX)' after expression
./murmur128_amd64.s:245: Error: junk `(BX)' after expression
./murmur128_amd64.s:245: Error: operand size mismatch for `movq'
```

**Comment by twmb** [comment]
Can you try https://github.com/twmb/murmur3/pull/12 and see if that fixes this?

**Comment by micwoj92** [comment]
Sorry for late response, it builds now. Thanks for the fix.

**Comment by twmb** [comment]
I'll see about releasing a patch for this today.

**Comment by twmb** [comment]
Released as v1.1.8, thanks for the report
